src/Native/System.Native/pal_runtimeinformation.h

```diff
@@ -14,5 +14,6 @@ enum
 {
     ARCH_X86,
     ARCH_X64,
-    ARCH_ARM
+    ARCH_ARM,
+    ARCH_ARM64
```

Comment:
On managed side, the ```s_osArch``` will remain null and is going to throw ```InvalidOperationException``` for arm64.

@manu-silicon Why do you want these changes right away? These can be made after the API-review, Abitlity to easily update the api surface area is one of the perks of an OOB assembly.